src/applications/debt-letters/app-entry.jsx

```diff
@@ -1,16 +1,16 @@
 import 'platform/polyfills';
 import './sass/debt-letters.scss';
-import routes from './routes';
+import Routes from './routes';
 
-import startApp from 'platform/startup';
+import startApp from 'platform/startup/router';
```

Comment:
Whats the reason for importing startApp from the platform router here?

Reply:
see https://dsva.slack.com/archives/C5HP4GN3F/p1626789649382300

Reply:
Based off of this [Slack thread](https://dsva.slack.com/archives/C5HP4GN3F/p1626789649382300) if you import the router with the first path that was removed it would import `react-router` but if it you add the `/router` it will import a router wrapped with `<BrowserRouter />` and give you access to the `react-router-dom`  API.

Reply:
Put a ticket in our backlog to document this information gap [here](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/department-of-veterans-affairs/va.gov-team/27666)